lib/Mojo/WebSocketProxy/Dispatcher.pm

```diff
@@ -61,7 +61,16 @@ sub open_connection {
 
     $config->{opened_connection}->($c) if $config->{opened_connection};
 
-    $c->on(json => \&on_message);
+    $c->on(json => sub {
+        my ($d, $hash) = @_;
+        on_message(@_) if $hash;
```

Comment:
what this condition means? What if it does not match? We silently ignore the result?